Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
We did extensively discuss this at one point. There is a little bit of a contention between trying to do a false start type of approach and having a SETUP ACK. The compromise was the LEASE behavior, which needed an ACK.